src/main/scala/io/iohk/ethereum/blockchain/sync/regular/BlockFetcherState.scala

```diff
@@ -179,10 +176,22 @@ case class BlockFetcherState(
       }
       .getOrElse(this)
 
+  def enqueueHeader(header: BlockHeader): BlockFetcherState = {
```

Comment:
please remove these methods and use safer counterparts - lack of them caused a bit of headache already